src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.cs

```diff
@@ -112,10 +118,10 @@ public void ResponseHeadersStart()
             WriteEvent(eventId: 11);
         }
 
-        [Event(12, Level = EventLevel.Informational)]
-        public void ResponseHeadersStop()
+        [Event(12, Level = EventLevel.Informational, Version = 1)]
+        public void ResponseHeadersStop(int statusCode)
```

Comment:
Do you want to differentiate between the name `statusCode` on `ResponseHeadersStop` and `RequestStop`?  Not strictly required, but I'm wondering if something like `intermediateStatusCode` would be appropriate here.  Totally a networking domain specific questions.

Reply:
In the majority of cases, the ones you see on Request vs RequestHeaders are one and the same, so my preference would be on naming them the same.